ipt/src/lib.rs

```diff
@@ -27,6 +27,9 @@ use sp_runtime::{
 };
 use sp_std::{convert::TryInto, vec::Vec};
 
+mod mock;
```

Comment:
I think it's better if you put `#[cfg(test)]` here and in the line below instead of in the respective files

Reply:
Thanks @arrudagates I've made the changes